### PR TITLE
Add new features to "Insert New Chart"

### PR DESCRIPTION
**Pull request by Wemmy0**
Minor change to add the line of best fit function into the 'Insert New Chart' form.

It's a minor change so possibly it can wait for the next version?

### CODE REVIEW

**Comment by Wemmy0** [comment]
I've added the `bestFit` modifiers to the modifiers section in the documentation and updated the README.md to represent the updated documentation.

**Comment by Wemmy0** [comment]
[obsidian-charts.zip](https://github.com/phibr0/obsidian-charts/files/9998744/obsidian-charts.zip)

**Comment by Wemmy0** [comment]
Here's the final:
[obsidian-charts.zip](https://github.com/phibr0/obsidian-charts/files/9998806/obsidian-charts.zip)

File: src/main.ts [hunk 1]
```diff
@@ -151,7 +151,7 @@ export default class ChartPlugin extends Plugin {
 		});
 
 		this.registerMarkdownCodeBlockProcessor('chart', this.postprocessor);
-		this.registerMarkdownCodeBlockProcessor('advanced-chart', async (data, el) => this.renderer.renderRaw(await JSON.parse(data), el));
+		this.registerMarkdownCodeBlockProcessor('obsidian-chart', async (data, el) => this.renderer.renderRaw(await JSON.parse(data), el));
```
Comment (phibr0) [inline src/main.ts:154]: This still needs to be changed!

**Comment by phibr0** [comment]
Thanks a lot!

**Pull request merged**